test/recipes/25-test_x509.t

```diff
@@ -116,11 +117,17 @@ sub test_errors { # actually tests diagnostics of OSSL_STORE
         $found = 1 if m/$expected/; # output must include $expected
     }
     close $in;
-    unlink $tmpfile;
+    # unlink $tmpfile;
```

Comment:
What is the intention here? Is this just temporarily commented out?

Reply:
At first I included the `unlink $tmpfile;` since usually tmp file should be cleaned up,
but as I recently learned from @levitte it's better not to do that for the files generated in `test-runs/test_*/`
because this makes debugging tests easier and those directories are temporary anyway.
So I made a comment out of this, which was meant to stay like this.

Or should I better add a comment like `# intentionally not doing unlink $tmpfile`?

Reply:
Yeah, perhaps something like `# $tmpfile is kept to help with investigation in case of failure`

Reply:
I like the suggested text; done.